docs/plugins.rst

```diff
@@ -170,6 +170,34 @@ Configuration in thumbor.conf:
 For monitors and keys, the values ​​used are those defined in the
 configuration file ceph.conf.
 
+
+`thumbor\_spaces <https://github.com/siddhartham/thumbor_spaces>`__ (by `Siddhartha Mukherjee <https://github.com/siddhartham>`__)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This plugin allows users to store objects in the DigitalOcean Spaces for
+result storage.
+
+-  *URL:* https://github.com/siddhartham/thumbor_spaces
+-  *Installing:* ``pip install thumbor_spaces``
+
+Using it is simple, just change your configuration in thumbor.conf:
+
+::
+
+    # Use rackspace for result storage.
```

Comment:
could you change `rackspace` here to `DigitalOcean Spaces`?

Reply:
done